### PR TITLE
Final changes

### PR DESCRIPTION
**Pull request by teekaytai**
In particular the Effort appendix

### CODE REVIEW

**Review by jhchee18 — COMMENTED** [review]
LGTM! Thanks for adding in the appendix 👍

**Review by eesung00 — COMMENTED** [review]
LGTM 👍

File: docs/diagrams/StatusClassDiagram.puml [hunk 1]
```diff
@@ -1,4 +1,8 @@
 @startuml
+!include style.puml
+skinparam classBackgroundColor MODEL_COLOR
+show members
+
 enum Status {
```
Comment (jolynloh) [inline docs/diagrams/StatusClassDiagram.puml]: do we need an ` <<enumeration>> ` label for the diagram?

**Review by jolynloh — COMMENTED** [review]
LGTM!

File: docs/diagrams/StatusClassDiagram.puml (hunk 1 above)
Comment (teekaytai) [reply docs/diagrams/StatusClassDiagram.puml]: Oh yeah probably should add that

**Pull request merged**